protobufs/livekit_rtc_v2.proto

```diff
@@ -91,9 +132,23 @@ message ConnectResponse {
   repeated Codec enabled_publish_codecs = 8;
   bool fast_publish = 9;
   SessionDescription subscriber_sdp = 10;
+  SessionDescription publisher_sdp = 11;
 }
 
-message ErrorResponse {
-  uint32 message_id = 1;
-  string error = 2;
+enum AddTrackState {
+  UNUSED = 0;
+  ADDED = 1;
+  QUEUED = 2;
+}
+
+message RPC_AddAudioTrackResponse {
+  RPC_AddAudioTrackRequest rpc_add_audio_track_request = 1; // request reflected back
```

Comment:
Reflecting the entire request for convenience on sender side. They have full context in the reply.